tests/chainer_tests/functions_tests/loss_tests/test_triplet.py

```diff
@@ -21,10 +21,13 @@
 class TestTriplet(unittest.TestCase):
 
     def setUp(self):
+        # Sample differentiable inputs
         eps = 1e-3
+        if self.dtype == numpy.float16:
+            eps *= 10
```

Comment:
Is there any reason you wrote this way instead of `eps = 1e-2`?

Reply:
No, either should be fine. Do you want me to change it?

Reply:
Yes, please.

Reply:
Fixed.